test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -270,6 +270,81 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | should be ([char]0x1b)
     }
 
+    Context "Test Unicode escape sequences." {
+        # These tests require the file to be saved with a BOM.  Unfortunately when this UTF8 file is read by
+        # PowerShell without a BOM, the file is incorrectly interpreted as ASCII.
+        It 'Test that Unicode escape sequence `u2195 in string returns the ↕ character.' {
+            $result = ExecuteCommand '"foo`u2195abc"'
+            $result | should be "foo↕abc"
+        }
+
+        It 'Test that Unicode escape sequence `u2195 in here string returns the ↕ character.' {
+            $result = ExecuteCommand ("@`"`n`n" + 'foo`u2195abc' + "`n`n`"@")
+            $result | should be "`nfoo↕abc`n"
+        }
+
+        It 'Test that the bracketed Unicode escape sequence `u{4} returns EOT control character.' {
+            $result = ExecuteCommand '"`u{4}"'
+            [int]$result[0] | should be 4

```

Comment:
Do you mean I should remove this test and add a ShouldBeParseError test? Because what you suggest would be a parse error.  Without braces, there must be 4 hex digits that follow `` `u``.